### ISSUE TITLE
Converts URLs to lower-case, which is not equivalent

### ISSUE THREAD

**Issue by rubenv** [k3a/html2text#9]
Consider this piece of code:

```go
package main

import (
	"fmt"

	"github.com/k3a/html2text"
)

func main() {
	result := html2text.HTML2Text(`Something <a href="http://bit.ly/2n4wXRs">http://bit.ly/2n4wXRs</a>`)
	fmt.Println(result)
	fmt.Printf("Correct? %v", result == `Something http://bit.ly/2n4wXRs`)
}
```

Which you can run here: https://play.golang.org/p/KpgQrOSkoyZ

This outputs the following:
```
Something http://bit.ly/2n4wxrs
Correct? false
```

For some reason the URL gets lower-cased, which is definitely not equivalent: the path part is case-sensitive.

**Comment by k3a** [comment]
Fixed in 1.0.5. Thanks for reporting it!

**Comment by rubenv** [comment]
Thanks for the quick fix! :champagne:
